src/main/java/org/broadinstitute/consent/http/service/DatasetRegistrationService.java

```diff
@@ -177,6 +196,21 @@ private List<FileStorageObject> uploadFiles(Map<String, FormDataBodyPart> files,
 
     }
 
+    private List<FileStorageObject> uploadFilesForStudy(Map<String, FormDataBodyPart> files,
```

Comment:
Minor naming nit. This method is extracting a list of Study FSOs from the mass of supplied form data. Something like `extractStudyFiles` might be more descriptive.

Reply:
I think these private methods could use some javadoc.